modules/sdk-coin-apt/src/lib/transaction/transaction.ts

```diff
@@ -39,6 +39,7 @@ export abstract class Transaction extends BaseTransaction {
   protected _feePayerSignature: Signature;
   protected _sender: string;
   protected _recipient: TransactionRecipient;
+  protected _recipients: TransactionRecipient[];
```

Comment:
initialized `_recipients` with empty list so that we don't have to add null check anywhere.